react/features/display-name/components/web/DisplayName.js

```diff
@@ -148,14 +158,18 @@ class DisplayName extends Component<Props, State> {
             allowEditing,
             displayNameSuffix,
             elementID,
+            _hasWallet,
+            _akAddress,
             t
         } = this.props;
+        const isNameDisabled = Boolean(_hasWallet && _akAddress);
```

Comment:
not sure about `_akAddress` but we can leave it as it is.

Reply:
I approved PR

Reply:
> not sure about `_akAddress` but we can leave it as it is.

I just afraid of that case:
- _akAddress is persisted somehow
- user load the page and can't change the name

Reply:
but it's not about the present version due we don't have persisted _akAddress

Reply:
@mike-gusev-ae I'm not sure that we need to persist that value in future.

Reply:
I think because it also have `_hasWallet` it's also require to use wallet so there is no problem in future that described at https://github.com/aeternity/jitsi-meet/pull/202#discussion_r475558252